httputil/httputil.go

```diff
@@ -12,20 +12,28 @@ const (
 	operationGet    = "GET"
 	operationPost   = "POST"
 	operationDelete = "DELETE"
+	operationPut = "PUT"
 )
 
-type HttpClient struct {
+type InbuiltHttpClient struct {
```

Comment:
Nit : InbuiltHttpClient  is a little weird. Since we are in the httputil package, can we name this "client" ?